kubernetes/README.md

```diff
@@ -17,9 +17,9 @@ Python 2.7 and 3.4+
 If the python package is hosted on Github, you can install directly from Github
 
 ```sh
-pip install git+https://github.com/kubernetes-client/python.git
+pip install git+https://github.com/kubernetes-kubernetes.client/python.git
 ```
-(you may need to run `pip` with root permission: `sudo pip install git+https://github.com/kubernetes-client/python.git`)
+(you may need to run `pip` with root permission: `sudo pip install git+https://github.com/kubernetes-kubernetes.client/python.git`)
```

Comment:
Please manually restore the commands in L20 and L22. I believe we have some misconfig of automation/swagger-codegen that writes the extra `kubernetes.`